ratatui-widgets/src/canvas/line.rs

```diff
@@ -124,8 +180,58 @@ mod tests {
     use crate::canvas::Canvas;
 
     #[rstest]
-    #[case::off_grid(&Line::new(-1.0, -1.0, 10.0, 10.0, Color::Red), ["          "; 10])]
-    #[case::off_grid(&Line::new(0.0, 0.0, 11.0, 11.0, Color::Red), ["          "; 10])]
+    #[case::off_grid(&Line::new(-1.0, 0.0, -1.0, 10.0, Color::Red), ["          "; 10])]
```

Comment:
The purpose of the case names (off_grid etc.) is to make it easy to work out which test failed without having to count down the index. Can you please add number suffixes?

Reply:
I add named suffixes instead of numbers. It's ok?

Reply:
The problem is that there are multiple cases with the same name. The names need to be unique to avoid having to count the cases to work out which of them failed.